.github/workflows/linkcheck.yml

```diff
@@ -14,7 +17,10 @@ jobs:
   linkcheck:
     name: Documentation linkcheck
     runs-on: ubuntu-latest
-    if: github.event_name != 'pull_request' || contains(github.event.pull_request.labels.*.name, 'perform linkcheck in CI')
+    if: >-
+      github.event_name == 'workflow_dispatch' ||
+      (github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'perform linkcheck in CI')) ||
+      (github.event_name == 'schedule' && github.repository == 'PlasmaPy/PlasmaPy' && github.ref == 'refs/heads/main')
```

Comment:
This is saying that this workflow should only be run:
 - If we manually trigger it (via workflow dispatch)
 - It's a pull request that has been labeled with "perform linkcheck in CI"
 - As a cron job if it's on the `main` branch of `PlasmaPy/PlasmaPy`.

My motivation here is to avoid cron jobs being run on forks, which can waste a lot of computing resources.

I'll be honest...ChatGPT helped me write this one.